extensions/youtube-downloader/src/index.tsx

```diff
@@ -182,13 +182,27 @@ export default function DownloadVideo() {
   useEffect(() => {
     (async () => {
       const clipboardText = await Clipboard.readText();
-      if (clipboardText && isYouTubeURL(clipboardText)) setValue("url", clipboardText);
+      if (clipboardText && isYouTubeURL(clipboardText)) {
+        setValue("url", clipboardText);
+        return;
+      }
 
-      const selectedText = await getSelectedText();
-      if (selectedText && isYouTubeURL(selectedText)) setValue("url", selectedText);
+      try {
+        const selectedText = await getSelectedText();
+        if (selectedText && isYouTubeURL(selectedText)) {
+          setValue("url", selectedText);
+          return;
+        }
+      } catch (e) {
+        console.log(e);
+      }
 
-      const tabUrl = (await BrowserExtension.getTabs()).find((tab) => tab.active)?.url;
-      if (tabUrl && isYouTubeURL(tabUrl)) setValue("url", tabUrl);
+      try {
+        const tabUrl = (await BrowserExtension.getTabs()).find((tab) => tab.active)?.url;
+        if (tabUrl && isYouTubeURL(tabUrl)) setValue("url", tabUrl);
+      } catch (e) {
+        console.error(e);
```

Comment:
Can we suppress these logs to the console? These are a bit annoying for developers.

Both line 197 and line 204 need to be removed.

Reply:
Done